cli/src/test/java/com/expediagroup/streamplatform/streamregistry/cli/command/delete/DeleteTest.java

```diff
@@ -0,0 +1,333 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.cli.command.delete;
+
+import static com.google.common.collect.ObjectArrays.concat;
+import static org.mockito.Mockito.inOrder;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import picocli.CommandLine;
+import picocli.CommandLine.ParseResult;
+
+import com.expediagroup.streamplatform.streamregistry.cli.command.delete.EntityClient.StreamKeyWithSchemaKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.ConsumerBindingKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.ConsumerKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.DomainKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.InfrastructureKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.ProducerBindingKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.ProducerKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.SchemaKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.StreamBindingKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.StreamKey;
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity.ZoneKey;
+
+@RunWith(MockitoJUnitRunner.Silent.class)
+public class DeleteTest {
+  private final CommandLine underTest = new CommandLine(new Delete());
+
+  private final String[] standardOptions = {"--bootstrapServers=bootstrapServers", "--schemaRegistryUrl=schemaRegistryUrl", "--streamRegistryUrl=streamRegistryUrl"};
+
+  private final DomainKey domainKey = new DomainKey("domain");
```

Comment:
This is optional, but since you're using the strings "domain", "schema", "stream", "producer" and "consumer" in several places, it would have been easier to declare them as static final fields instead of hardcoding it :D

Reply:
done